source/scenario/map/Grid.gd

```diff
@@ -4,100 +4,81 @@ class_name Grid
 var rect := Rect2()
 var map: TileMap = null
 
+var base_hex_size = Vector2(1, sqrt(3)/2)
```

Comment:
where does this come from?
The terrain sprites in Wesnoth are 72x72, the tiles themselves are 72x54. that's (1, 0.75) not (1, 0.86).
But maybe I am missing something?

Reply:
Ah sorry nevermind that. It was some leftover of some debug code I added when I was trying to parse the code :) I removed it now